src/server/events/index.ts

```diff
@@ -124,10 +124,10 @@ export async function addOrUpdateEvent(
 }
 
 export async function getCompanyEvents(
-	companyID: string,
+	companyID: ObjectID,
```

Comment:
Why replace type here?

Reply:
db IDs should always be objectID if possible, doesn't make sense to be string?

Reply:
react hooks can't return ObjectIDs in queries, frontend components don't have ObjectID nor should they - this function then can't be used in frontend components?

Reply:
yeah this is a serverside function though